tmt/cli.py

```diff
@@ -530,6 +530,41 @@ def finito(
         click_context.obj.run.go()
 
 
+def _apply_linters(lintable: Union[
+            tmt.lint.Lintable[tmt.base.Test], 
+            tmt.lint.Lintable[tmt.base.Plan], 
+            tmt.lint.Lintable[tmt.base.Story], 
+            tmt.lint.Lintable[tmt.base.LintableCollection]], 
+            linters: list[tmt.lint.Linter],
+            failed_only: bool,
+            enforce_checks: list[str],
+            outcomes: list[tmt.lint.LinterOutcome],) -> tuple[bool, Union[list[tmt.lint.LinterRuling], None]]:
```

Comment:
* one spurious comma after `outcomes` type,
* instead of `Union[foo, None]`, use `Optional` - it's the same, but we use `Optional` consistently throughout the code.

Reply:
Changed :heavy_check_mark: